packages/fetchai/connections/p2p_libp2p/check_dependencies.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
relatively simple; we can iterate over it to add finer-grained checks, e.g. check the version is at least greater than some version etc.